pdf.go

```diff
@@ -17,6 +115,11 @@ func ConvertPDF(r io.Reader) (string, map[string]string, error) {
 	}
 	defer f.Done()
 
+	// Verify if pdf has images or is pdf only-text
+	if PDFHasImage(f.Name()) {
```

Comment:
Yes, in our case, we use this check for PDFs typically generated by scanners, many of them generate PDF's with each page being the first photo of the original document.